src/ast/function.rs

```diff
@@ -92,6 +92,8 @@ pub enum Function {
     },
     #[strum(to_string = "UNWRAP")]
     Unwrap { expr: Expr, selector: Expr },
+    #[strum(to_string = "RANDOM_UUID")]
+    RandomUuid(),
```

Comment:
Q: Do you use random and uuid together?

Looking at the [documentation](https://en.wikipedia.org/wiki/Universally_unique_identifier#Version_4_(random)), it seems that v4(in UUID) already contains the meaning of random.